lib/core/BlogPostLayout.js

```diff
@@ -16,46 +16,66 @@ class BlogPostLayout extends React.Component {
   renderSocialButtons() {
     const post = this.props.metadata;
 
-    const fbLike = this.props.config.facebookAppId ? (
-      <div
-        className="fb-like"
-        data-layout="standard"
-        data-share="true"
-        data-width="225"
-        data-show-faces="false"
-      />
-    ) : null;
+    const fbComment = this.props.config.facebookAppId &&
+      this.props.config.facebookComment && (
+        <div>
+          <div
+            className="fb-comments"
+            data-href={
```

Comment:
The default value for `data-href` is the current page and it seems to be what we need. Do we need this prop/attr?

Reply:
in development the current url will fallback to `localhost` & cause an error, but in production it will work fine. Should we keep it for easy debugging. WDYT ?

Reply:
Oh I see. Let's keep it 😄